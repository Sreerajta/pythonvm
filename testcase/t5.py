print "numbers from 1 to 10 using while loop"

i=0
while(i<=10):
 print i
 i=i+1
