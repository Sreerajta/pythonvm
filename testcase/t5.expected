numbers from 1 to 10 using while loop
0
1
2
3
4
5
6
7
8
9
10
